internal/provider/resource_managed_identity_alias.go

```diff
@@ -94,9 +95,18 @@ func (t *managedIdentityAliasResource) Schema(_ context.Context, _ resource.Sche
 				Computed:            true,
 			},
 			"alias_source_id": schema.StringAttribute{
-				MarkdownDescription: "ID of the managed identity being aliased",
-				Description:         "ID of the managed identity being aliased",
-				Required:            true,
+				MarkdownDescription: "ID of the managed identity being aliased.",
+				Description:         "ID of the managed identity being aliased.",
+				Optional:            true,
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"alias_source_path": schema.StringAttribute{
+				MarkdownDescription: "Full path of the managed identity being aliased.",
+				Description:         "Full path of the managed identity being aliased.",
+				Optional:            true,
```

Comment:
Shouldn't this be computed as well?

Reply:
Is this true for values which are only used as input and won't be set on the resource?

Reply:
Hmm, I guess not. I didn't realize that we don't return the path in the response.